docs/kubernetes/KubernetesIntegration.md

```diff
@@ -6,19 +6,26 @@ This guide will describe the configuration required to use the Calico network pl
    The Calico network plugin looks for five environment variables. If one is not set, it will assume a default value. If you need to override the defaults, you must set these variables in the environment of the _kubelet_ process.
 
 * #####ETCD_AUTHORITY
-   By default, the Calico network plugin will assume that the etcd datastore is located at `<MASTER_IP>:6666`. Setting the `ETCD_AUTHORITY` variable in your environment will direct Calico to the correct IP if your cluster is set up differently.
-
-* #####CALICOCTL_PATH
```

Comment:
I don't think we should remove CALICOCTL_PATH entirely - we should indicate that it is deprecated in version `v0.4.0` and explain its behavior in previous versions.

There are still people using the previous version of the plugin which support this.